blog/2024-07-07-openhab-4-2-release.md

```diff
@@ -0,0 +1,415 @@
+- Florian ([@florian-h05](https://github.com/florian-h05)) for the openHAB Main UI
+- Jimmy ([@jimtng](https://github.com/jimtng)) for the openHAB Main UI
+- Laurent ([@lolodomo](https://github.com/lolodomo)) for the openHAB Basic UI
+
+Many of our top committers are also maintainers, which means besides contributing code, they also review pull requests and participate in discussions on GitHub issues, which helps maintain our high level of quality across bindings, UIs and Core.
+At present, we have 36 maintainers across all our repositories.
+Huge thanks to all of them for keeping a high development pace for openHAB 4.2!
+
+# Highlights
+
+As you might have already noticed in the activity statistics, there was much development on openHAB - too much to list everything.
+Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.2.0) to check out what's new and noteworthy (e.g. breaking changes).
+
+In the following sections, our maintainers and contributors are introducing some new features you don't want to miss!
+
+## Mobile Notification Enhancements
+
+_Dan Cunningham ([@digitaldan](https://github.com/digitaldan)), openHAB Maintainer_
+
+With openHAB 4.2 we have updated our mobile push notifications architecture, bringing modern, cutting edge notification features to both our Android and iOS clients.
+
+First, we have consolidated both mobile platforms to use Google Firebase Messaging, which greatly improves the reliability of push notifications from our cloud service and reduces the amount of work needed to add new features in a cross-platform way.
+
+This change has allowed us to add an incredible amount of new features to our app in a relatively short amount of time.
+
+### Notification Display Enhancements
+
+Notifications now contain many more additional features, such as:
+
+- Title
+  - Replaces the default "openHAB" text with a user supplied version
+- Media Attachments
+  - Add images (or a short video on iOS) from an external URL or openHAB image item
+- On Click Action
+  - Navigate to a specific sitemap, page, popup widget, etc... in our UIs when the notification is clicked
+- Action Buttons
+  - Add up to 3 buttons that allow for executing commands, navigating through our UI's or open links in an embedded browser
+
+<p align="center"><img style="max-width: 70%;" src="/uploads/2024-07-07-openhab-4-2-release/notifications.png"/></p>
+
+### Update / Collapse Notifications
+
+We have also added some really useful ways to manage notifications already pushed to a device through rules and actions!
+
+- Collapsible Notifications
+  - Instead of seeing 100 messages that you left the garage door open, have a messages replace the existing shown one, so only the latest message is show on a user's device.
+  - Update existing messages with new information, add additional actions, media, etc.. without showing multiple notifications.
```

Comment:
```suggestion
  - Instead of seeing 100 messages that you left the garage door open, have a new message replace the existing one shown, so only the latest message is visible on a user's device.
  - Update existing messages with new information, add additional actions, updated media, etc.. without showing multiple notifications.
```